### PR TITLE
Profile link fix

### LINKED ISSUE #3080: [FEATURE] If profile only has links then do not show dropdown or tab
### Description

Currently, a dropdown is rendered on mobile regardless of if it is needed or not.  This issue is to handle not rendering a dropdown for profiles that contain only links and only render a dropdown on 2 or more items.

### Screenshots

Current mobile view with links only, dropdown has been clicked in this image. Dropdown is not needed here
Screenshot Amanda profile on android with dropdown clicked showing it has one item only

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by arun-kushwaha04**
## Fixes Issue
Closes #3080 

## Changes proposed
Rendered a simple text in place of drop-down when user has only one tab present.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots
Screenshot from 2023-02-14 20-01-38

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4821"></a>

### CODE REVIEW

File: components/user/UserTabs.js [hunk 1]
```diff
@@ -2,152 +2,158 @@ import { BiSortAlt2 } from "react-icons/bi";
...
+  setUserData({ ...userData });
+ };
 
-  const getNested = (obj, args) => {
-    return args.reduce((obj, level) => obj && obj[level], obj);
-  };
+ const getNested = (obj, args) => {
+  return args.reduce((obj, level) => obj && obj[level], obj);
+ };
+ console.log(tabs);
```
Comment (Aadarsh805) [inline components/user/UserTabs.js:84]: remove this console.log

File: components/user/UserTabs.js [hunk 2]
```diff
@@ -2,152 +2,158 @@ import { BiSortAlt2 } from "react-icons/bi";
...
+       className="block w-full rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
+       defaultValue={tabs.find((tab) => tab.current).name}
+      >
+       {tabs.map((tab) => (
+        <option key={tab.name}>{tab.name}</option>
+       ))}
+      </select>
+     </>
+    ) : (
+     <h3 className="font-medium border-b-2 text-center">
```
Comment (Aadarsh805) [inline components/user/UserTabs.js:106]: Also add some padding on here, the border under looks too close

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Please check your formatting, it looks like you have changed every line, then when the next person puts the file back to the project formatting standards, they will then again change every line and it will be difficult to see the changes

File: components/user/UserTabs.js (hunk 2 above)
Comment (arun-kushwaha04) [reply components/user/UserTabs.js:106]: Sure

**Comment by arun-kushwaha04** [comment]
> Please check your formatting, it looks like you have changed every line, then when the next person puts the file back to the project formatting standards, they will then again change every line and it will be difficult to see the changes

@eddiejaoude Fixed the formatting issue.

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
All good here, you just need to remove the `package-lock.json` from this commit

**Review by Aadarsh805 — DISMISSED** [review]
yep good now

**Review by devvsakib — DISMISSED** [review]

File: components/user/UserTabs.js [hunk 3]
```diff
@@ -81,73 +81,79 @@ export default function UserTabs({ tabs, setTabs, setUserData, userData }) {
   const getNested = (obj, args) => {
     return args.reduce((obj, level) => obj && obj[level], obj);
   };
+ 
+ return (
+  <div>
+   <div className="sm:hidden">
+    {tabs.length > 1 ? (
```
Comment (eddiejaoude) [inline components/user/UserTabs.js:88]: I think this could be added above the `div` and therefore remove the need for the `<>` below

Comment (arun-kushwaha04) [reply components/user/UserTabs.js]: Sorry @eddiejaoude for late response, had collage exams. I have made the required changes.

**Comment by eddiejaoude** [comment]
Thank you for updating. 

Please...
- follow the coding standards in your IDE
- only make the changes you need to, otherwise it is very difficult to see your actual changes

File: components/user/UserTabs.js [hunk 4]
```diff
@@ -1,153 +1,157 @@
-import { BiSortAlt2 } from "react-icons/bi";
-import Link from "../Link";
+import { BiSortAlt2 } from 'react-icons/bi';
+import Link from '../Link';
```
Comment (eddiejaoude) [inline components/user/UserTabs.js:2]: Please allow our coding standards, these changes are not required and make the project inconsistent

**Comment by eddiejaoude** [comment]
Thank you for your contribution. But there are too many changes in this pull request it makes it difficult to see the changes for this specific issue. 

As there has been no activity from the author for over 4 weeks, I will close this pull request

**Pull request closed without merging**